cmd/nephe-controller/main.go

```diff
@@ -141,6 +155,11 @@ func main() {
 		&webhook.Admission{Handler: &nephewebhook.SecretValidator{Client: mgr.GetClient(),
 			Log: logging.GetLogger("webhook").WithName("secret")}})
 
+	// Register webhook for configmap.
+	mgr.GetWebhookServer().Register("/validate-v1-configmap",
+		&webhook.Admission{Handler: &nephewebhook.ConfigMapValidator{Client: mgr.GetClient(),
+			Log: logging.GetLogger("webhook").WithName("configmap"), Config: opts.config, NpController: *npController}})
```

Comment:
Have a PR out to keep the component name consistent for logging.
```suggestion
			Log: logging.GetLogger("webhook").WithName("ConfigMap"), Config: opts.config, NpController: *npController}})
```